certora/harness/StakePoolHarness.sol

```diff
@@ -77,4 +69,13 @@ contract StakePoolHarness is StakePool {
         ClaimRequest memory req = claimReqs[msg.sender][index];
         return _canBeClaimed(req);
     }
+
+    function transferOut(
```

Comment:
add a comment with the reasoning behind keeping this here in the harness

Reply:
Done